RELEASE/scripts/sl_ascend.ash

```diff
@@ -11364,16 +11368,8 @@ boolean LX_handleSpookyravenFirstFloor()
 		{
 			expectPool += 3;
 		}
-		if(possessEquipment($item[Pool Cue]) && sl_is_valid($item[Pool Cue]) && !have_equipped($item[Pool Cue]) && (expectPool < 18))
-		{
-			equip($slot[weapon], $item[Pool Cue]);
-		}
-		if(have_equipped($item[Pool Cue]))
-		{
-			expectPool += 3;
-		}
 
-		if(!possessEquipment($item[Pool Cue]))
+		if(!possessEquipment($item[Pool Cue]) && !posessEquipment($item[7964]) && !posessEquipment($item[2268]) && !posessEquipment($item[7961]))
```

Comment:
Can you add inline comments saying what the items referred to by number are, just for easier future reference?

Reply:
I can do one better, and write self-documenting code